tests/integrations/config/test_config_client.rs

```diff
@@ -80,3 +82,101 @@ fn test_dispatch_change() {
     // config change should also dispatch to raftstore config manager
     assert_eq!(mgr.0.lock().unwrap().raft_log_gc_threshold, 2000);
 }
+
+#[test]
+fn test_write_update_to_file() {
```

Comment:
How about also testing something like `compression-per-level`?

Reply:
`compression-per-level` has value of array which are not support to change dynamically for now, but I added testing for `compression-per-level` at unit test `test_toml_writer`, PTAL.